packages/node_modules/@webex/helper-image/src/index.js

```diff
@@ -49,18 +49,24 @@ export function readExifData(file, buf) {
       file &&
       (file.type === 'image/jpeg' || file.mimeType === 'image/jpeg')
     ) {
-      const {
-        Orientation,
-        ExifImageHeight,
-        ExifImageWidth
-      } = await parse(buf, {translateValues: false});
+      try {
+        const {
+          Orientation,
+          ExifImageHeight,
+          ExifImageWidth
+        } = await parse(buf, {translateValues: false});
 
-      file.orientation = Orientation;
-      file.exifWidth = ExifImageWidth;
-      file.exifHeight = ExifImageHeight;
+        file.orientation = Orientation;
+        file.exifWidth = ExifImageWidth;
+        file.exifHeight = ExifImageHeight;
 
-      if (file.image) {
-        file.image.orientation = Orientation;
+        if (file.image) {
+          file.image.orientation = Orientation;
+        }
+      }
+      catch (error) {
+        // Error handle if parse doesn't detect any EXIF data
+        console.error('Helper-Image#readExifData --> exifr error: ', error);
```

Comment:
should this be sent to the logger instead?

Reply:
if i replace it with the if statement i probably won't be logging the error, as it wasn't logging before i replaced the package